### PR TITLE
Sort datasets within megasets

### PR DESCRIPTION
**Pull request by meganrm**
Problem
=======
Megaset sub cards are getting randomly ordered

Solution
========
I sorted the datasets based on title 

## Type of change
Please delete options that are not relevant.

* Bug fix (non-breaking change which fixes an issue)

Steps to Verify:
----------------
1. npm start
2. the main variance dataset should be first

### CODE REVIEW

File: src/components/MegasetCard/index.tsx [hunk 1]
```diff
@@ -27,6 +27,8 @@ const MegasetCard: React.FunctionComponent<MegasetCardProps> = ({
     // container instead of the container expanding to fit publications
     const maxWidth = numDatasets > 1 ? "100%" : datasetCardWidth;
 
+    // basic assumption here is that shorter titles may indicate primacy 
+    const datasets = reverse(sortBy(map(megaset.datasets), "title"))
```
Comment (toloudis) [inline src/components/MegasetCard/index.tsx:31]: the comment says shorter titles but is this an alphabetical order string sort?

Comment (toloudis) [reply src/components/MegasetCard/index.tsx:31]: Why not just add some kind of metadata array that defines the order so that the title can be anything?

Comment (meganrm) [reply src/components/MegasetCard/index.tsx:31]: ok yeah, we can add that on the data side.... my understanding of this issue based on the slack comments was that the issue was the randomness more than the actual order so this seemed like a good solution to fix that problem

Comment (meganrm) [reply src/components/MegasetCard/index.tsx]: Regarding the alphabetical sort: it was reverse alphabetical. I just meant that if you have `dataset` and `dataset_another_thing`, we'd want to have the dataset first. I just made a change that makes it clearer

**Comment by toloudis** [comment]
I don't mind approving this sorting because it fixes the randomness but it's not fully solving #125 as it's not likely to display the datasets in the order that the stakeholders want to see.

**Review by toloudis — APPROVED** [review]

**Pull request merged**